src/Analyzers/shared/FeatureDetection/Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I see there is one more thing you have to do. You'll need to change to explicit top/bottom imports like this:

```xml
<Project>
  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />

  rest of project stuff goes here

  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />

  <!-- This project does not produce a binary. Disable the compiler. -->
  <Target Name="Compile" />
  <Target Name="CopyFilesToOutputDirectory" />
</Project>
```